src/lib.rs

```diff
@@ -259,6 +261,29 @@ mod contract {
         result.map(|_f| Vec::new()).sdk_process();
     }
 
+    #[no_mangle]
+    pub extern "C" fn register_relayer() {
+        let relayer_address = sdk::read_input();
+        let mut engine = Engine::new(predecessor_address());
+        engine.register_relayer(
+            sdk::predecessor_account_id().as_slice(),
+            Address(relayer_address.as_slice().try_into().unwrap()),
+        );
+    }
+
+    /// Allow receiving NEP141 tokens to the EVM contract
+    #[no_mangle]
+    pub extern "C" fn ft_on_transfer() {
+        let predecessor_account_id = String::from_utf8(sdk::predecessor_account_id()).unwrap();
+        let current_account_id = String::from_utf8(sdk::current_account_id()).unwrap();
```

Comment:
Nit: these probably don't need to be strings if all we are going to do is compare them on the next line.

Reply:
@birchmd this is only a template function at the moment. The strings will be used in the implementation (when me merge this PR into our PRs)

Reply:
https://github.com/aurora-is-near/aurora-engine/pull/62#discussion_r627506198